src/sql/src/parsers/tql_parser.rs

```diff
@@ -116,7 +118,7 @@ impl<'a> ParserContext<'a> {
             }
             _ => ("0".to_string(), "0".to_string(), "5m".to_string(), None),
         };
-        let query = Self::parse_tql_query(parser, self.sql)?;
+        let query = Self::parse_tql_query(parser, self.sql).context(ParserSnafu {})?;
```

Comment:
Looks like `.context(ParserSnafu)?` is more simple. The brackets `{}` are unnecessary.

Can you remove such brackets in other places too? Thank u.

Reply:
sure, will adjust shortly!